.travis.yml

```diff
@@ -0,0 +1,40 @@
+language: go
+# see https://docs.travis-ci.com/user/reference/overview/#Virtualization-environments
+# for the detail
+# sudo: requried
+dist: trusty
+env:
+  global:
+    - REGISTRY_USER=nfvperobot
+    - secure: "ca+qSuPGFyg75kspazh0W21sk/0Eag8jD5Hwq8br0mr3KMt/OvujEbvvEekHIS1i8gn27NQPHKQxWMF2ntZr6obD28GfOc88w4SaWJDp4lFR4x2VvpKdPIqHKVxt3DRwJ4YKLH3E70QL4bJAh4ScxI0AeKB3EDCh4wOwEWF7AfjgKNfNFxoJa5arfdd1nfGfGjHEArHO39YY5uXZmdWGEQ/QyCDJy39RQcrqwvfErM6GXoO43cmRJkzEh45A5M/bdZ+6wpR/LKevp2ftWprOref4Y9GevJoEtDzZb47CXMZSmjB7JJUj9vWUiGrGm0TSZwJfOsw07Fs1ZzE5IHgSwdPldVn+XUCQICJY7pS3mAnyJ+lwCnrzTvKYd46V08JJ/VJzfZFlgbsygiMdPMypfEZ/0Ll51MXKFH3mPP2KYxjkgUZA8CAPvSWpMeLLAO+neSVas+CC5+C3pfaBEYGbLGZXim7Jtdxw2nVeOKaKQLZn/c3W+zrTbWV/obYOfUH5x0XS9IA/J6aUm4M3tq3n0SVGIk3EKi7fF4kESf0xW+6IyKEFs8k1tynQhWZ7i4gTwQ9ug+XyIm6HWGfUz3YkZVkLQhHDfwJ9DSYvXO3seWigY1GkY+dDys5Nb+loiR1Y0mKrDFdTw91GIpde3ayys/b+Klj0RYY17CWuQEh+9NA="
+
+before_install:
+  - sudo apt-get update -qq
+
+install:
+  - go get github.com/golang/lint/golint
+
+before_script:
+  - golint .
+  - go fmt .
+  - go vet .
+  # was failing with: `gocyclo: vendor/golang.org/x/net/context/go19.go:15:14: expected type, found '=' (and 1 more errors)`
+  # - gocyclo -over 15 .
+
+script:
+  - go build dummy.go
```

Comment:
just do go build here and generate a binary called k8s-dummy-device-plugin. Same for other places where we build and run binary.

Reply:
cool, updated the name of the resulting binary with the `-o output_binary_name_here` option for `go build`